src/server.c

```diff
@@ -3287,11 +3288,23 @@ void resetCommandTableStats(void) {
         c = (struct redisCommand *) dictGetVal(de);
         c->microseconds = 0;
         c->calls = 0;
+        c->failed_calls = 0;
     }
     dictReleaseIterator(di);
 
 }
 
+void resetErrorTableStats(void) {
+    raxIterator ri;
+    raxStart(&ri,server.errors);
+    raxSeek(&ri,"^",NULL,0);
+    while(raxNext(&ri)) {
+        struct redisError *err = ri.data;
+        if(err) err->count=0;
```

Comment:
i think we never expect this to be NULL, let i rather drop the `if` and let it crash if we're wrong.
```suggestion
        err->count=0;
```